File: javascript_programs/cambioRealDolar.js
let real = parseFloat(prompt("Digite uma quantia em R$: "))
let cambio = parseFloat(prompt("Cotação do dólar hoje: "))

let dolar = real / cambio
console.log(`R$${real.toFixed(2)} está valendo hoje US$${dolar.toFixed(2)}`)

File: python_programs/cambio_real_dolar.py
real = float(input("Digite uma quantia em R$: "))
cambio = float(input("Cotação do dólar hoje: "))
dolar = real / cambio
print("R$", real, "está valendo hoje US$", dolar)
print(f'R${real:.2f} está valendo hoje US${dolar:.2f}') #F-Strings
